multiqc/modules/adapterRemoval/adapterRemoval.py

```diff
@@ -0,0 +1,311 @@
+                    self.len_dist_plot_data['mate2'][self.s_name][l_data[0]] = 0
+                    self.len_dist_plot_data['singleton'][self.s_name][l_data[0]] = 0
+                    self.len_dist_plot_data['collapsed'][self.s_name][l_data[0]] = 0
+                    self.len_dist_plot_data['collapsed_truncated'][self.s_name][l_data[0]] = 0
+                    self.len_dist_plot_data['discarged'][self.s_name][l_data[0]] = l_data[2]
+                    self.len_dist_plot_data['all'][self.s_name][l_data[0]] = l_data[3]
+                else:
+                    # this case should not be reached (see case at method set_ar_type())
+                    pass
+            else:
+                if not self.__collapsed:
+                    self.len_dist_plot_data['mate1'][self.s_name][l_data[0]] = l_data[1]
+                    self.len_dist_plot_data['mate2'][self.s_name][l_data[0]] = l_data[2]
+                    self.len_dist_plot_data['singleton'][self.s_name][l_data[0]] = l_data[3]
+                    self.len_dist_plot_data['collapsed'][self.s_name][l_data[0]] = 0
+                    self.len_dist_plot_data['collapsed_truncated'][self.s_name][l_data[0]] = 0
+                    self.len_dist_plot_data['discarged'][self.s_name][l_data[0]] = l_data[4]
+                    self.len_dist_plot_data['all'][self.s_name][l_data[0]] = l_data[5]
+                else:
+                    self.len_dist_plot_data['mate1'][self.s_name][l_data[0]] = l_data[1]
+                    self.len_dist_plot_data['mate2'][self.s_name][l_data[0]] = l_data[2]
+                    self.len_dist_plot_data['singleton'][self.s_name][l_data[0]] = l_data[3]
+                    self.len_dist_plot_data['collapsed'][self.s_name][l_data[0]] = l_data[4]
+                    self.len_dist_plot_data['collapsed_truncated'][self.s_name][l_data[0]] = l_data[5]
+                    self.len_dist_plot_data['discarged'][self.s_name][l_data[0]] = l_data[6]
+                    self.len_dist_plot_data['all'][self.s_name][l_data[0]] = l_data[7]
+
+    def adapter_removal_stats_table(self):
+
+        headers = OrderedDict()
+        headers['percent_aligned'] = {
+                'title': '% Trimmed',
+                'description': '% trimmed reads',
+                'max': 100,
+                'min': 0,
+                'suffix': '%',
+                'scale': 'RdYlGn',
+                'format': '{:.1f}%',
+                'shared_key': 'percent_aligned',
+        }
+        headers['aligned_total'] = {
+                'title': 'Total Trimmed',
+                'description': 'total trimmed reads',
+                'min': 0,
+                'scale': 'PuBu',
+                'shared_key': 'aligned_total'
+        }
+        self.general_stats_addcols(self.adapter_removal_data, headers)
+
+    def adapter_removal_retained_chart(self):
+
+        pconfig = {
+            'title': 'retained and discarded',
```

Comment:
Please use Sentence Case. Also, perhaps `Discarded Reads` would be a little clearer?